dashboard/app/models/script_level.rb

```diff
@@ -258,7 +258,20 @@ def next_level_or_redirect_path_for_user(
       # to that lesson
       script_lesson_extras_path(script.name, (extras_lesson || lesson).relative_position)
     else
-      level_to_follow ? build_script_level_path(level_to_follow) : script_completion_redirect(script)
+      # To help teachers have more control over the pacing of certain
+      # scripts, we send students on the last level of a lesson to the unit
+      # overview page.
+      if end_of_lesson? &&
+        script.should_show_unit_overview_between_lessons &&
+        user.has_pilot_experiment?('end-of-lesson-redirects')
```

Comment:
Hitting https://studio.code.org/experiments/set_single_user_experiment/end-of-lesson-redirects will add you to the experiment.  Right now I have it set up locally for my own testing and on production.